test/fuzz_rdflib.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import atheris
+
+with atheris.instrument_imports():
+    from rdflib import Graph
+    import sys
+    from xml.sax import SAXParseException
+
+
+def TestOneInput(data):
+    if len(data) < 20:
+        return
+    fdp = atheris.FuzzedDataProvider(data)
+
+    format_list = ["xml", "trix", "turtle", "n3", "nt", ]
+    g = Graph()
+    try:
+        g.parse(format=fdp.PickValueInList(format_list),
+                data=fdp.ConsumeUnicodeNoSurrogates(100))
+    except (SAXParseException, IndexError, Exception):
+        pass
```

Comment:
Having `Exception` here may be over broad, as that includes the other two errors and potentially errors that should not occur.

I think even IndexError may constitute an error, we should possibly consider what Exceptions make sense, and possibly clean up the exceptions raised by RDFLib to be more sane.

Reply:
Agreed, I have changed the fuzz test to only except issues with inappropriate inputs (as far as I can tell - let me know if you think they could also be bugs)